core/src/wallet/common/OperationQuery.cpp

```diff
@@ -85,6 +85,9 @@ namespace ledger {
             case api::OperationOrderKey::BLOCK_HEIGHT:
                 _builder.order("height", std::move(descending), "b");
                 break;
+            case api::OperationOrderKey::TIME:
+                _builder.order("time", std::move(descending), "b"); // NOLINT
```

Comment:
i know and I tried itn but `.order()` takes a `bool &&` as a 2nd parameter... that's the reason of the move... It's useless I know, that's something we might refactor, but maybe separately to avoid adding even more changes to this PR ?